docs/intro/validator.rst

```diff
@@ -0,0 +1,225 @@
+=========
+Validator
+=========
+
+Getting started
+===============
+Validator is an instrument to unify the process of model training and validation.
+There are two main reasons to wrap your model using classes from `Validator` submodule:
+
+- make the structure of the model lifecycle clear,
+- provide API to automized model training and validation.
+
+All you need is:
+
```

Comment:
There are several sub-processes.
A model developer has to create a Validator subclass, but he might have nothing to do with the validation process.
While an acceptor has to define validation procedure and thus configure yaml file or even several files.